bitnami/clickhouse/values.yaml

```diff
@@ -35,12 +35,15 @@ global:
       ## @param global.compatibility.openshift.adaptSecurityContext Adapt the securityContext sections of the deployment to make them compatible with Openshift restricted-v2 SCC: remove runAsUser, runAsGroup and fsGroup and let the platform use their allowed default IDs. Possible values: auto (apply if the detected running cluster is Openshift), force (perform the adaptation always), disabled (do not perform adaptation)
       ##
       adaptSecurityContext: auto
+
 ## @section Common parameters
-##
 
 ## @param kubeVersion Override Kubernetes version
 ##
 kubeVersion: ""
+## @param apiVersions Override Kubernetes API versions reported by .Capabilities
+##
```

Comment:
Missing these values:

```
existingExtraUsersConfigmap
extraUsersFiles
keeper.existingExtraUsersConfigmap
keeper.extraUsersFiles
tls.client.existingSecret
```

I see them in the template but not in the values.yaml file